tests/transformations/subtests/transpile-js/parcel/index.md

```diff
@@ -2,4 +2,4 @@
 result: pass
 ---
 
-ES5 is the default in Parcel.
+By default, Parcel doesn't transpile. Setting a browserslist config (i.e. in `package.json#browserslist`) will enable transpilation to support the specified browsers (though for await/async, [another Babel plugin has to be specified](https://v2.parceljs.org/languages/babel/#extending-the-default-babel-config)).
```

Comment:
This special configuration for async/await isn't necessary anymore with beta 3, but there's also a bug in the await to generator downleveling.

Should I change the test code? Or link the bug and mark as partial?

Reply:
Any idea when a release will be made with the fix?

Reply:
The next planend release (RC) is in about one month.

(The fix itself is https://github.com/parcel-bundler/parcel/pull/6325)

Reply:
I'm happy to wait until then, rather than turn a pass into a partial.

Reply:
It took more than one month of course, but I've bumped the version and it's working now.